recipes-graphics/wayland/weston-init/weston-waitforcard.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
+
+DEFAULT_CARD=/dev/dri/card0
```

Comment:
weston-start allows scripts in /usr/share/weston-start/* to be sourced before launching Weston. Would it be feasible for a script there to check what backend Weston is using and verify that such a device exists? It'd be pretty much the same as waitforcard.sh, except for FB/DRM and so on, and in another location.